conf/provisioning/alerting/sample.yaml

```diff
@@ -0,0 +1,165 @@
+# # config file version
+apiVersion: 1
+
+# # List of rule groups to import or update
+# groups:
+#     # <int> organization id, default = 1
+#   - orgId: 123
+#     # <string, required> name of the rule group
+#     name: my_rule_group
+#     # <string, required> name of the folder the rule group will be stored in
```

Comment:
This can be name? I thought it had to be UID? or does something do the translation?

Reply:
I put a translation layer in-between to make it more comfortable for the user.